docs/use/setup-wallet/index.md

```diff
@@ -48,7 +48,7 @@ On Ethereum, the system supports [MetaMask](https://metamask.io/download.html),
 
```

Comment:
Line 41: image
![](./images/wallet-component1.png#width=40%;)

Change with this one 
![image](https://github.com/centrifuge/documentation/assets/90972248/f405f9cf-deda-4d08-ab3e-21f9cbd1ab12)